index.js

```diff
@@ -52,7 +55,7 @@ MockServerResponse.prototype.writeHead = function(statusCode, reason, headers) {
 };
 
 MockServerResponse.prototype._getString = function() {
-	return Buffer.concat(this._readableState.buffer).toString();
```

Comment:
@diachedelic I see this has been merged already, but it seems to `.toString()` was removed by accident.

When I apply these changes in my node_modules version (`0.2.1` installed via `mock-express-response`) I get this as a result:
`<Buffer 3c 21 44 4f 43 54 59 50 ... >`

When adding the `.toString();` I'm getting the correct string result.